From ee731e30f0e76eff4a0ea4acf27165ee241125ae Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Alex=20R=C3=B8nne=20Petersen?= <alex@alexrp.com>
Date: Sun, 19 Oct 2025 02:02:19 +0200
Subject: [PATCH 016/133] delete file accidentally added in
 6de339d5d3deb4c7a1585288888f8e788e489eb6
X-Developer-Signature: v=1; a=openpgp-sha256; l=267; i=xtex@astrafall.org;
 h=from:subject; bh=jxoLBpH8PCjwcLrmJ53ZnCTJW1qx5PMUAFpoqQIqTwc=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDJn7cn99P9X75/eeWUu7NPZkLBRKM+y4sNf1jtcU+43+U
 /8HWRi96ZjIwiDGxWAppshSZNjgzaqTzi+6rFwWZg4rE8gQaZEGBgYGBhYGvtzEvFIjHSM9U21D
 PUNDHSMdIwYuTgGY6l5/RobmpUqOHjGWjMtyxLkMvr/0U7TJWXXUMV3WJPTO9guswRMYGbZNbd5
 ocns6t8OcqWWLJl361RO5b2XajBcqt5vUC4Od/ZgB
X-Developer-Key: i=xtex@astrafall.org; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

(cherry picked from commit c37d23f45ae6bd0db6b072180d7b84566c7dc8a2)
---
 crt.c | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 delete mode 100644 crt.c

diff --git a/crt.c b/crt.c
deleted file mode 100644
index e69de29bb2d1..000000000000
-- 
2.52.0

